src/components/ContactSection.jsx

```diff
@@ -0,0 +1,198 @@
+import React, { useRef } from 'react'
+import { useForm } from 'react-hook-form';
+import { MailIcon } from '@heroicons/react/outline'
+
+const ContactSection = () => {
+    const nameRef = useRef()
+    const emailRef = useRef()
+    const telephoneRef = useRef()
+    const messageRef = useRef()
+    const countryRef = useRef()
+    const dioseceRef = useRef()
+
+    // const handleSubmit = (e) => {
+    //     e.preventDefault()
+    //     /* 
```

Comment:
You can always remove these unused code just for clarity.

Reply:
Also landing page would look a lot better if margins are consistent through out every section.